src/main/java/Wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,12 @@
+package Wolox.training.repositories;
+
+import java.io.Serializable;
+import org.springframework.data.repository.*;
+import org.springframework.beans.annotation.*;
+
+@NoRepositoryBean
```

Comment:
I think you pushed the old, not working, version. 
Update the repository to the new version